### PR TITLE
Move API version to v1

### PR DESCRIPTION
**Pull request by darkowlzz**
This changes the API version of all the CRDs from v1alpha1 to v1.

### CODE REVIEW

File: deploy/storageos-operators.configmap.yaml [hunk 1]
```diff
@@ -463,7 +463,7 @@ data:
 
     - packageName: storageos
       channels:
-      - name: alpha
+      - name: stable
         currentCSV: storageosoperator.0.0.0
```
Comment (croomes) [inline deploy/storageos-operators.configmap.yaml]: Is `0.0.0` correct?

**Comment by darkowlzz** [comment]
`deploy/examples.json` and `deploy/olm/storageos/` still have `v1alpha1` api version because they are related to the OLM CSV file which target a specific version of the operator. They should be updated once we have a v1 api version release available.

File: deploy/storageos-operators.configmap.yaml (hunk 1 above)
Comment (darkowlzz) [reply deploy/storageos-operators.configmap.yaml]: yes, this is used in the e2e test. We need not increment it every time.

**Comment by actionbuddha** [comment]
Adding @avestuk  and @Arau for commentary and awareness - guys, be aware this is a breaking change.

**Comment by domodwyer** [comment]
I noticed the `Default Value` for `images.nodeContainer` on the readme says `storageos/node:1.1.0` - I guess this should be `1.1.4` and changing it part of the release checklist @actionbuddha ?

**Pull request merged**